source/sass/library/components/dropdown-field.scss

```diff
@@ -0,0 +1,36 @@
+.dropdown-field {
```

Comment:
I created this path because there's really nowhere to add pattern library components from a sass perspective rather than a jsx or wagtail standpoint. I figured since we're expanding the style guide, this made sense 🤷‍♀

Reply:
If we are changing the default `<select>` element style, we can just override rules for `select` without creating a new dropdown class like `.dropdown-field`.

e.g., 

`select`
`select.large` etc